rust/noosphere-ns/src/server/handlers.rs

```diff
@@ -1,112 +1,134 @@
 use crate::{Multiaddr, NSRecord, NameSystem, NameSystemClient, NetworkInfo, Peer, PeerId};
-use anyhow::{Error, Result};
+use anyhow::Result;
+use axum::response::{IntoResponse, Response};
 use axum::{extract::Path, http::StatusCode, Extension, Json};
 use noosphere_core::data::Did;
-use std::future::Future;
 use std::sync::Arc;
 use tokio::sync::Mutex;
-use tracing::*;
+
+pub struct JsonErr(StatusCode, String);
+impl IntoResponse for JsonErr {
+    fn into_response(self) -> Response {
+        let mut res = Json(Err::<String, String>(self.1)).into_response();
```

Comment:
Note the `Result<String>` object to JSONify responses: `{ "Err": "something went awry" }`